4-docker-swarm/README.md

```diff
@@ -131,7 +131,7 @@ Let's kill one of the worker nodes and see how docker re-schedules its container
 
 Now `docker service ps pinger` repeatedly to see how some of the pop up in the other nodes automatically. How cool is that?
 
-You now have a resilient application running distributedly in a docker swarm cluster ✨
+You now have a resilient application running distributed in a docker swarm cluster ✨
```

Comment:
`distributedly` es correcto, pero agree que suena raro. Lo cambiás a "... resilient, distributed application running in ..."?